.circleci/config.yml

```diff
@@ -3,9 +3,86 @@ version: 2
 jobs:
   build:
     docker:
-      - image: ubuntu:16.04
+      - image: googleapis/curl:latest
     steps:
       - run:
-          name: No-op.
-          command: echo "This command did nothing. It exists to configure CircleCI so that PRs that supply actual configuration will work."
-    working_directory: /var/code/googleapis
+          name: Add command to be able to trigger jobs.
+          command: |
+            touch trigger_job.sh && chmod a+x trigger_job.sh
```

Comment:
Every command is independent; the function definition would not survive to the next step.